package.json

```diff
@@ -5,15 +5,15 @@
     "doc": "docs"
   },
   "engines": {
-    "pnpm": "7.6.0"
+    "pnpm": ">=7.6.0"
```

Comment:
we don't really want to leave anything loose/open-ended. In the past, this has caused differences between developers' machines, which then become difficult to track down and fix.

Reply:
If you want to upgrade to the latest `pnpm` - please do that in the same PR and rephrase your commit messages/PR title.

Reply:
I have my local version of this file with no `pnpx` and allows my version of pnpm, so I am fine with that.

I think having to do a PR every time there is a `pnpm` release is overkill,  specially since they seem to release frequently, but suit yourselves :) 

I'll close this PR